### PR TITLE
Create A Secure Ubuntu 18.04 Snapshot on Hetzner Cloud

### PR DESCRIPTION
**Pull request by rericsson**
Hi,

I've written a draft of a tutorial that shows how to create an Ubuntu snapshot that meets the DevSec Linux Baseline specification. I hope it meets your requirements and look forward to any comments to improve the content. 

I have read and understood the Contributor's Certificate of Origin available at the end of
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Robert Ericsson rob@l10systems.com

### CODE REVIEW

File: tutorials/create-secure-ubuntu-18-04-snapshot/01.en.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+SPDX-License-Identifier: MIT
+path: "/tutorials/create-secure-ubuntu-18-04-snapshot"
+slug: "create-secure-ubuntu-18-04-snapshot"
+date: "2019-10-05"
+title: "Create A Secure Ubuntu 18.04 Snapshot on Hetzner Cloud"
+short_description: "This tutorial has step-by-step instructions to create a Ubuntu 18.04 snapshot hardened to the DevSec Linux Baseline that can be used as a secure base for Ubuntu servers."
+tags: ["Cloud", "Ubuntu", "Security"]
+author: "Rob Ericsson"
+author_link: "https://github.com/rericsson"
+author_img: "https://avatars3.githubusercontent.com/u/4484005?s=400&v=4"
```
Comment (n-se) [inline tutorials/create-secure-ubuntu-18-04-snapshot/01.en.md]:
```suggestion
author_img: "https://avatars3.githubusercontent.com/u/4484005"
```

File: tutorials/create-secure-ubuntu-18-04-snapshot/01.en.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+## Introduction
+
+Security is a challenge to implement properly. This tutorial shows how to leverage [Hashicorp Packer](https://www.packer.io/), [Chef](https://www.chef.sh/) and [Inspec](https://www.inspec.io/) to create a snapshot of baseline Ubuntu 18.04 that adheres to the [DevSec Hardening Framework](https://dev-sec.io/) in Hetzner Cloud. This snapshot can be used to create new secured servers in Hetzner Cloud for your use.  
+
+**Prerequisites**
+
+To follow the steps in this tutorial, you will need the following:
+
+* A basic understanding of Hetzner Cloud. You should know what a server and snapshot are and how they relate to each other.
+* You should have an active Hetzner Cloud account and an API token generated on your Hetzner project.
```
Comment (n-se) [inline tutorials/create-secure-ubuntu-18-04-snapshot/01.en.md]: Maybe you link here to https://community.hetzner.com/tutorials/howto-hcloud-cli

File: tutorials/create-secure-ubuntu-18-04-snapshot/01.en.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+
+First, [download](https://www.packer.io/downloads.html) the appropriate package for your system. Packer is provided as a zip file. 
+
+Unzip the downloaded package into a directory where Packer will be installed. On Linux, ~/packer or /usr/local/bin is generally good, depending on whether you want to restrict the install to just your user or install it system-wide. On Windows systems, you can put it wherever you'd like. The packer (or packer.exe for Windows) binary is all that is necessary to run Packer. 
+
+The final step to installation is to make sure the directory you installed Packer to is on the PATH. See [this](https://stackoverflow.com/questions/14637979/how-to-permanently-set-path-on-linux-unix) for instructions on setting the PATH on Linux. [This](https://stackoverflow.com/questions/1618280/where-can-i-set-path-to-make-exe-on-windows) has instructions for setting the PATH on Windows.
+
+Test your install by opening a command line and typing:
+
+```packer --version```
```
Comment (n-se) [inline tutorials/create-secure-ubuntu-18-04-snapshot/01.en.md]: For a single line code block better use only one ` at the beginning and end

File: tutorials/create-secure-ubuntu-18-04-snapshot/01.en.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+(c) The contribution was provided directly to me by some other person
+    who certified (a), (b) or (c) and I have not modified it.
+
+(d) I understand and agree that this project and the contribution are
+    public and that a record of the contribution (including all personal
+    information I submit with it, including my sign-off) is maintained
+    indefinitely and may be redistributed consistent with this project
+    or the license(s) involved.
+
+Signed-off-by: Rob Ericsson rob@l10systems.com
```
Comment (n-se) [inline tutorials/create-secure-ubuntu-18-04-snapshot/01.en.md]: You need to close the comment here too with `-->` (check the template if you don't know how)

**Comment by rericsson** [comment]
Great. Thanks for the review. 

Rob

**Comment by leojakeson** [comment]
Hello Rob, we prefer not to have links to stackoverflow questions. Please remove those two links in step 1, and instead provide your own explanation.

Once you have done that we can merge this request.

**Comment by rericsson** [comment]
Removed the links and committed the changes. 

Rob

File: tutorials/create-secure-ubuntu-18-04-snapshot/01.en.md [hunk 5]
```diff
@@ -37,13 +37,23 @@ First, [download](https://www.packer.io/downloads.html) the appropriate package
...
+The final step to installation is to make sure the directory you installed Packer to is on the PATH. 
+
+To set the path on Linux, run the following:
+
+`export PATH="$PATH:/usr/local/bin"`
+
+or 
+
+`export PATH="$PATH:~/packer"`
+
```
Comment (n-se) [inline tutorials/create-secure-ubuntu-18-04-snapshot/01.en.md]: You should also add this somehow to your bashrc, if you just export it in your current session it wont be persistent in the next sessions.

Comment (rericsson) [reply tutorials/create-secure-ubuntu-18-04-snapshot/01.en.md]: OK. I updated the instruction to update .profile or .bash_profile if you have updated the PATH. Makes sense.

**Comment by rericsson** [comment]
Any other edits or is this ready to go? Thanks!

**Pull request merged**